src/battle_script_commands.c

```diff
@@ -337,262 +337,262 @@ static void Cmd_metalburstdamagecalculator(void);
-	Cmd_painsplitdmgcalc, // 0xA5
-	Cmd_settypetorandomresistance, // 0xA6
-	Cmd_setalwayshitflag, // 0xA7
-	Cmd_copymovepermanently, // 0xA8
-	Cmd_trychoosesleeptalkmove, // 0xA9
-	Cmd_setdestinybond, // 0xAA
-	Cmd_trysetdestinybondtohappen, // 0xAB
-	Cmd_settailwind, // 0xAC
-	Cmd_tryspiteppreduce, // 0xAD
-	Cmd_healpartystatus, // 0xAE
-	Cmd_cursetarget, // 0xAF
-	Cmd_trysetspikes, // 0xB0
-	Cmd_setforesight, // 0xB1
-	Cmd_trysetperishsong, // 0xB2
-	Cmd_handlerollout, // 0xB3
-	Cmd_jumpifconfusedandstatmaxed, // 0xB4
-	Cmd_handlefurycutter, // 0xB5
-	Cmd_setembargo, // 0xB6
-	Cmd_presentdamagecalculation, // 0xB7
-	Cmd_setsafeguard, // 0xB8
-	Cmd_magnitudedamagecalculation, // 0xB9
-	Cmd_jumpifnopursuitswitchdmg, // 0xBA
-	Cmd_setsunny, // 0xBB
-	Cmd_maxattackhalvehp, // 0xBC
-	Cmd_copyfoestats, // 0xBD
-	Cmd_rapidspinfree, // 0xBE
-	Cmd_setdefensecurlbit, // 0xBF
-	Cmd_recoverbasedonsunlight, // 0xC0
-	Cmd_setstickyweb, // 0xC1
-	Cmd_selectfirstvalidtarget, // 0xC2
-	Cmd_trysetfutureattack, // 0xC3
-	Cmd_trydobeatup, // 0xC4
-	Cmd_setsemiinvulnerablebit, // 0xC5
-	Cmd_clearsemiinvulnerablebit, // 0xC6
-	Cmd_setminimize, // 0xC7
-	Cmd_sethail, // 0xC8
-	Cmd_jumpifattackandspecialattackcannotfall, // 0xC9
-	Cmd_setforcedtarget, // 0xCA
-	Cmd_setcharge, // 0xCB
-	Cmd_callterrainattack, // 0xCC
-	Cmd_cureifburnedparalysedorpoisoned, // 0xCD
-	Cmd_settorment, // 0xCE
-	Cmd_jumpifnodamage, // 0xCF
-	Cmd_settaunt, // 0xD0
-	Cmd_trysethelpinghand, // 0xD1
-	Cmd_tryswapitems, // 0xD2
-	Cmd_trycopyability, // 0xD3
-	Cmd_trywish, // 0xD4
-	Cmd_settoxicspikes, // 0xD5
-	Cmd_setgastroacid, // 0xD6
-	Cmd_setyawn, // 0xD7
-	Cmd_setdamagetohealthdifference, // 0xD8
-	Cmd_setroom, // 0xD9
-	Cmd_tryswapabilities, // 0xDA
-	Cmd_tryimprison, // 0xDB
-	Cmd_setstealthrock, // 0xDC
-	Cmd_setuserstatus3, // 0xDD
-	Cmd_assistattackselect, // 0xDE
-	Cmd_trysetmagiccoat, // 0xDF
-	Cmd_trysetsnatch, // 0xE0
-	Cmd_trygetintimidatetarget, // 0xE1
-	Cmd_switchoutabilities, // 0xE2
-	Cmd_jumpifhasnohp, // 0xE3
-	Cmd_getsecretpowereffect, // 0xE4
-	Cmd_pickup, // 0xE5
-	Cmd_docastformchangeanimation, // 0xE6
-	Cmd_trycastformdatachange, // 0xE7
-	Cmd_settypebasedhalvers, // 0xE8
-	Cmd_jumpifsubstituteblocks, // 0xE9
-	Cmd_tryrecycleitem, // 0xEA
-	Cmd_settypetoterrain, // 0xEB
-	Cmd_pursuitrelated, // 0xEC
-	Cmd_snatchsetbattlers, // 0xED
-	Cmd_removelightscreenreflect, // 0xEE
-	Cmd_handleballthrow, // 0xEF
-	Cmd_givecaughtmon, // 0xF0
-	Cmd_trysetcaughtmondexflags, // 0xF1
-	Cmd_displaydexinfo, // 0xF2
-	Cmd_trygivecaughtmonnick, // 0xF3
-	Cmd_subattackerhpbydmg, // 0xF4
-	Cmd_removeattackerstatus1, // 0xF5
-	Cmd_finishaction, // 0xF6
-	Cmd_finishturn, // 0xF7
-	Cmd_trainerslideout, // 0xF8
-	Cmd_settelekinesis, // 0xF9
-	Cmd_swapstatstages, // 0xFA
-	Cmd_averagestats, // 0xFB
-	Cmd_jumpifoppositegenders, // 0xFC
-	Cmd_trygetbaddreamstarget, // 0xFD
-	Cmd_tryworryseed, // 0xFE
-	Cmd_metalburstdamagecalculator, // 0xFF
+    Cmd_attackcanceler, // 0x0
```

Comment:
Can you align the number comments all to the right so they're in the same column? Upstream seems to do that so we should match that.